lib/please_cli/please_cli/deploy.py

```diff
@@ -463,3 +463,117 @@ def cmd_TASKCLUSTER_HOOK(ctx,
         output,
         ask_for_details=interactive,
     )
+
+
+@click.command()
+@cli_common.cli.taskcluster_options
+@click.argument(
+    'project',
+    required=True,
+    type=click.Choice(please_cli.config.PROJECTS),
+    )
+@click.option(
+    '--channel',
+    type=click.Choice(please_cli.config.CHANNELS),
+    envvar="GITHUB_BRANCH",
+    required=True,
+    )
+@click.option(
+    '--nix-build',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix-build',
+    help='`nix-build` command',
+    )
+@click.option(
+    '--nix',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix',
+    help='`nix` command',
+    )
+@click.option(
+    '--docker-repo',
+    required=True,
+    default=please_cli.config.DOCKER_REPO,
+    help='Docker repository.',
+    )
+@click.option(
+    '--image-tag-format',
```

Comment:
maybe lets add `--docker-` in front to make it obvious this is about docker image.
if turn this option to list of templates (`multiple=True`) and if you add `{hash}` and `{version}` as template parameters you could have a default that would do exactly what you hard-coded below.